app/steps/ui/deceased/dob/template.html

```diff
@@ -35,17 +35,20 @@
             {
                 classes: "govuk-input--width-2" + (" govuk-input--error" if fields["dob-day"].error or fields["dob-date"].error),
                 name: "day",
-                value: fields["dob-day"].value
+                value: fields["dob-day"].value,
+                autocomplete: "dob-day"
```

Comment:
Autocomplete attribute is 'bday-day', similarly with month and year

Reply:
Removed this as it doesn't make sense to have an autocomplete on the DoB of someone who died :)